roles/cockpit/tasks/check.yml

```
@@ -17,7 +17,7 @@
     state: "started"
     enabled: true
     daemon_reload: true
-  when: result.failed is true
+  when: result|failed
```

Tested again still failed. This worked:

```suggestion
  when: result.failed
```